pkg/kv/kvserver/batcheval/cmd_add_sstable.go

```diff
@@ -37,6 +38,14 @@ func init() {
 	RegisterReadWriteCommand(roachpb.AddSSTable, DefaultDeclareIsolatedKeys, EvalAddSSTable)
 }
 
+var addSSTableRewriteConcurrency = settings.RegisterIntSetting(
```

Comment:
Can we make this metamorphic?

Reply:
I was going to leave it as zero for now until I do some more production tests?

Reply:
added